### ISSUE TITLE
reloading web page in browser with cache bypass (ctl-r) only reconstruct cache for current zoom level

### ISSUE THREAD

**Issue by tomhughes** [tomhughes/trac-tickets#5410]
**[Submitted to the original trac issue database at 7.38pm, Sunday, 14th August 2016]**

If the user reloads osm web page in browser with cache bypass (ctl-r), the page is reloaded and the new edits are visible.

Unfortunatly, if the user zooms in/out, the page still reflects the cache and the last edits are not shown.

The user expect that all the zoom levels are regenerated with a single ctl-r

**Comment by tomhughes** [comment]
**Author: TomH**
**[Added to the original trac issue at 7.49pm, Sunday, 14th August 2016]**

Well ctrl-R doesn't bypass the cache anyway - that is a normal reload.

I assume you mean ctrl-shift-R which does bypass the cache. That is a browser feature and it applies to the resources loaded by the page at that moment in time. There is no way for us to make it apply to resources loaded in the future as you manipulate the zoom. In any case how long would you expect the effect to last? One minute? One hour? One day?

In short this is impossible, so we won't be doing it I'm afraid.
